### PR TITLE
add rbac roles for opsportal

### PR DESCRIPTION
**Pull request by jr0d**
https://jira.d2iq.com/browse/D2IQ-64340

adds roles for:
 /ops/portal/kommander/ui
/ops/portal/kommander/monitoring/grafana
/ops/portal/kommander/monitoring/query
/ops/portal/kommander/monitoring/grafana

### CODE REVIEW

**Comment by alejandroEsc** [comment]
Do these roles belong here or in their respective charts? Could we get more context please?

File: stable/kommander/values.yaml [hunk 1]
```diff
@@ -132,3 +133,19 @@ grafana:
...
+      traefik.ingress.kubernetes.io/priority: "2"
+      traefik.ingress.kubernetes.io/auth-type: forward
+      traefik.ingress.kubernetes.io/auth-url: http://traefik-forward-auth-kubeaddons.kubeaddons.svc.cluster.local:4181/
+      traefik.ingress.kubernetes.io/auth-response-headers: X-Forwarded-User
+    path: /ops/portal/kommander/ui
+
+portalRBAC:
+  grafana:
+    enabled: true
+  kommanderUserInterface:
```
Comment (samvantran) [inline stable/kommander/values.yaml]: nit: maybe shorten to `kommanderUI`?

File: stable/kommander/templates/grafana/ingress-roles.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - nonResourceURLs:
+      - {{ .Values.grafana.ingress.path | trimSuffix "/"}}
+      - {{ .Values.grafana.ingress.path | trimSuffix "/" }}/*
+    verbs:
+      - get
+      - head
+      - post
+      - put
+      - patch
+  {{- end}}
```
Comment (samvantran) [inline stable/kommander/templates/grafana/ingress-roles.yaml]: nit: should this line be unindented? same q for the others

File: stable/kommander-karma/values.yaml [hunk 3]
```diff
@@ -80,3 +80,6 @@ karma:
 
   certSecretNames:
     - kommander-karma-client-tls
+
+portalRBAC:
+  enabled: true
```
Comment (samvantran) [inline stable/kommander-karma/values.yaml]: newline

Comment (shaneutt) [reply stable/kommander-karma/values.yaml]:
```suggestion
  enabled: true
  
```

File: stable/kommander/templates/grafana/ingress-roles.yaml (hunk 2 above)
Comment (jr0d) [reply stable/kommander/templates/grafana/ingress-roles.yaml]: :+1:

File: stable/kommander/values.yaml (hunk 1 above)
Comment (jr0d) [reply stable/kommander/values.yaml]: I don't like `kommanderUI` looks for some reason.

File: stable/kommander-karma/values.yaml (hunk 3 above)
Comment (jr0d) [reply stable/kommander-karma/values.yaml]: :+1:

File: stable/kommander/values.yaml (hunk 1 above)
Comment (samvantran) [reply stable/kommander/values.yaml]: okay 👍

File: stable/kommander/templates/grafana/ingress-roles.yaml (hunk 2 above)
Comment (samvantran) [reply stable/kommander/templates/grafana/ingress-roles.yaml]: looks like this is still indented

File: stable/kommander-karma/templates/ingress-roles.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  - nonResourceURLs:
+      - {{ .Values.karma.ingress.path | trimSuffix "/"}}
+      - {{ .Values.karma.ingress.path | trimSuffix "/" }}/*
+    verbs:
+      - get
+      - head
+      - post
+      - put
+      - patch
+  {{- end}}
```
Comment (samvantran) [inline stable/kommander-karma/templates/ingress-roles.yaml]: here too

File: stable/kommander/templates/grafana/ingress-roles.yaml (hunk 2 above)
Comment (jr0d) [reply stable/kommander/templates/grafana/ingress-roles.yaml]: I don't know how I missed that

**Comment by jr0d** [comment]
had to rebase... which  auto-dismissed reviews

**Comment by juliangieseke** [comment]
included and merged in #471

**Pull request closed without merging**